src/app/services/data.service.ts

```diff
@@ -0,0 +1,172 @@
+      date: '11.07.2019',
+      time: '14:01:01',
+      author: 'Vitali',
+      source: 'The inthernet'
+    },
+    {
+      id: '6',
+      title: 'Text 6',
+      shortDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing',
+      content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto animi architecto voluptatum sapiente natus ipsam eos, totam magnam corrupti facilis?',
+      date: '11.09.2019',
+      time: '14:01:01',
+      author: 'John K.',
+      source: 'the inthernet'
+    },
+    {
+      id: '7',
+      title: 'Text 7',
+      shortDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing',
+      content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto animi architecto voluptatum sapiente natus ipsam eos, totam magnam corrupti facilis?',
+      date: '11.09.2019',
+      time: '19:01:01',
+      author: 'Smith R.',
+      source: 'CNN'
+    },
+    {
+      id: '8',
+      title: 'Text 8',
+      shortDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing',
+      content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto animi architecto voluptatum sapiente natus ipsam eos, totam magnam corrupti facilis?',
+      date: '11.03.2019',
+      time: '14:03:01',
+      author: 'no name',
+      source: 'CNN'
+    },
+    {
+      id: '9',
+      title: 'Text 9',
+      shortDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing',
+      content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto animi architecto voluptatum sapiente natus ipsam eos, totam magnam corrupti facilis?',
+      date: '11.02.2019',
+      time: '10:01:01',
+      author: 'Vitali',
+      source: 'The inthernet'
+    },
+    {
+      id: '10',
+      title: 'Text 10',
+      shortDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing',
+      content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto animi architecto voluptatum sapiente natus ipsam eos, totam magnam corrupti facilis?',
+      date: '11.03.2019',
+      time: '14:04:01',
+      author: 'John',
+      source: 'BBC'
+    }
+  ];
+
+  public newsSource = new BehaviorSubject<object>(this.newsItems.slice(0, 5));
+  public currentNews = this.newsSource.asObservable();
+
+  constructor(private router: Router) { }
+
+  public delete(listItems: News[] | undefined, id: string): void {
+      const deleteNews = listItems.find((item: News) => item.id === id);
+      listItems.splice(listItems.indexOf(deleteNews), 1);
+      this.newsSource.next(listItems);
+      this.router.navigate(['/main']);
+  }
+
+  public loadNews(currentItems: News[]): void {
+    this.newsSource.next(currentItems.concat(this.newsItems.slice(5)));
+  }
+
+  public getItem(id: string): News {
+    return this.newsItems.find((item: News) => item.id === id);
+  }
+
+  public filterByMe(): void {
+    this.newsSource.next(this.newsItems.filter((item: News) => item.author === 'Vitali'));
```

Comment:
string literal use in enum and across all app, if you change in another place of app this string you should change string in all places. with enum you can change only in one place and no fear to break app